### ISSUE TITLE
Showcase snapshots/certificate pending on doc website

### ISSUE THREAD

**Issue by jpraynaud** [input-output-hk/mithril#315]
Requires #324 to be completed

A first version could showcase:
  * The `Pending Certificate` if it exists, and the list of the latest `Snapshots` on a first page
  * The `Snapshots` provides a link to the associated `Certificate` details on a new page
  * The `Certificate` provides a link to the `Previous Certificate` in the chain if it exists

Nice to have:
* Use for the demo with the `devnet` in local website
* Have a refresh every `30s` on the first page
* Responsive design
